ebitenutil/loadimage_go_1_16.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2014 Hajime Hoshi
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build (darwin || freebsd || js || linux || windows) && !android && !ios && go1.16
+// +build darwin freebsd js linux windows
+// +build !android
+// +build !ios
+// +build go1.16
+
+package ebitenutil
+
+import (
+	"image"
+	"io/fs"
+
+	"github.com/hajimehoshi/ebiten/v2"
+)
+
+// NewImageFromFS loads the file from a fs.FS filesystem with path and returns ebiten.Image and image.Image.
+//
+// go1.16+ only
+//
+// Image decoders must be imported when using NewImageFromFS. For example,
+// if you want to load a PNG image, you'd need to add `_ "image/png"` to the import section.
+//
+// This is useful if you've ebedded a filesystem with embed.FS.
+func NewImageFromFS(filesystem fs.FS, path string) (*ebiten.Image, image.Image, error) {
```

Comment:
Hmm, while `NewImageFromReader` is fine, I tend not to want to have this.

```
f, err := filesystem.Open(path)
if err != nil {
    return err
}
defer f.Close()
img, _, err := ebitenutil.NewImageFromReader(f)
if err != nil {
    return err
}
// Use img
```

Is this so burden?

Reply:
I agree it has less value than `NewImageFromReader` (which only has a bad alternative). I'll remove it from this PR.